src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,312 @@
+          <img
+            src={mode == LIGHT_MODE ? AlertCircle : AlertCircle2}
+            alt="Alert Icon"
+            style={{
+              width: "18px",
+              height: "18px",
+            }}
+          />
+        </Flex>
+
+        {/* right total visit section  */}
+        <Flex alignItems="center" gap="30px">
+          <Box
+            css={{
+              width: "61px",
+              border: "2px solid $secondaryLight",
+              borderRadius: "$lg",
+            }}
+          ></Box>
+
+          <Text
+            size="xs"
+            weight={"medium"}
+            css={{
+              color: "$textSecondary",
+            }}
+          >
+            Provisions Month
+          </Text>
+
+          <Popover>
+            <PopoverTrigger asChild>
+              <Button
+                css={{
+                  fontSize: "14px",
+                  color: "$textPrimary",
+                  fontWeight: "400",
+                  border: "$borderWidths$xs solid $border",
+                  padding: "$5 !important",
+                  lineHeight: "$xs",
+                  "& svg": {
+                    height: "$4 !important",
+                    width: "$4 !important",
+                  },
+                }}
+                size={"md"}
+                variant={"ghost"}
+                rightIcon={<CalendarIcon size={10} />}
+              >
+                {FormattedDateFn()}
+              </Button>
+            </PopoverTrigger>
+            <PopoverContent
+              css={{
+                width: "auto",
+              }}
+            >
+              <CalendarRange
+                onChange={(e) => {
+                  setTimeout(() => {
+                    setValue(e);
+                    handleDateRangeChange(e.start, e.end);
+                  }, 100);
+                }}
+                minValue={parseDate("2023-07-10")}
+              />
+              <PopoverArrow />
+            </PopoverContent>
+          </Popover>
+
+          <DropdownMenu>
+            <DropdownMenuTrigger asChild>
+              <Flex
+                css={{
+                  padding: "8px",
+                  border: "$borderWidths$xs solid $border",
+                  borderRadius: "5px",
+                  cursor: "pointer",
+                }}
+              >
+                <img
+                  src={mode == LIGHT_MODE ? HorizontalDots : HorizontalDotsDark}
+                  alt="Dots"
+                  style={{
+                    width: "14px",
+                    height: "14px",
+                  }}
+                />
+              </Flex>
+            </DropdownMenuTrigger>
+            <DropdownMenuContent
+              alignOffset={0}
+              sideOffset={5}
+              align="end"
+              css={{
+                cursor: "pointer",
+              }}
+            >
+              <DropdownMenuItem onClick={() => handleCountChange(null)}>
+                All
```

Comment:
@9xVibee can you move these to constants as array of objects and map the values here